appveyor/test_task.bat

```diff
@@ -91,6 +91,11 @@ if not exist "%PHP_BUILD_CACHE_ENCHANT_DICT_DIR%\en_US.aff" (
 mkdir %USERPROFILE%\enchant\myspell
 copy %PHP_BUILD_CACHE_ENCHANT_DICT_DIR%\* %USERPROFILE%\enchant\myspell
 
+set TEST_PHPDBG_EXECUTABLE=%PHP_BUILD_OBJ_DIR%\Release
+if "%THREAD_SAFE%" equ "1" set TEST_PHPDBG_EXECUTABLE=%TEST_PHPDBG_EXECUTABLE%\_TS
+set TEST_PHPDBG_EXECUTABLE=%TEST_PHPDBG_EXECUTABLE%\phpdbg.exe
+if "%OPCACHE%" equ "1" set TEST_PHPDBG_EXECUTABLE=
```

Comment:
On Linux TEST_PHPDBG_EXECUTABLE gets auto-detected in run-tests.php. Maybe it should be on Windows as well?

Reply:
There is already some detection:
https://github.com/php/php-src/blob/4f47ba99f002d50e11c111b8625d81f79b2bf52f/run-tests.php#L180
but this assumes packaged builds (`nmake snap`). Before the builds are packaged, the executable is in a folder which depends on architecture, TS and Debug/Release – unless the user configured a custom output folder. Not sure that it's worth to support all this.

Reply:
Is it in the same directory as the main PHP executable though? In that case the `dirname($php) . "/phpdbg"` check should cover it, if extended to `.exe`.

Reply:
Yes, it is in the same directory. The problem is that `nmake test` passes `-p` to run-tests.php to set the proper executable, but the code to detect `TEST_PHPDBG_EXECUTABLE` is executed before the options are set. :(